### PR TITLE
Add sequence numbers etc

### PR DESCRIPTION
**Pull request by ed-sparkes**
## Description

Add some additional logging to understand what is going on 

## Type of change

Delete not appropriate

- New feature (non-breaking change which adds functionality)

## Testing

Please tick the testing that has been completed

- [x] Unit tests
- [ ] Integration tests

## Developer Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have run the [code formatting checks](../README.md#code-quality)
- [ ] I have run the [code quality checks](../README.md#code-quality)
- [ ] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the team
- [ ] Unit test code coverage is at or above 80%
- [ ] New and existing unit tests pass locally with my changes
- [ ] Tests have added that prove my fix is effective or that my feature works (Integration tests)
- [ ] I have made corresponding changes to the documentation
- [ ] I have cleaned down my environment (if created)

## Code Reviewer Checklist

- [ ] I have run the unit tests and they run correctly
- [ ] I can confirm the changes have been tested or approved by a tester
- [ ] I can confirm no remaining infrastructure is left over from this branch

### CODE REVIEW

**Review by mabe13 — APPROVED** [review]

**Review by rohith1122 — APPROVED** [review]
LGTM

File: application/event_processor/changes.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def update_changes_with_address(changes: dict, dos_service: DoSService, nhs_uk_e
     dos_address = dos_service.address
 
     if dos_address != nhs_uk_address_string:
-        logger.debug(f"Address is not equal, {dos_address=} != {nhs_uk_address_string=}")
+        logger.info(f"Address is not equal, {dos_address=} != {nhs_uk_address_string=}")
```
Comment (JackPlowman) [inline application/event_processor/changes.py:68]: Why is address info level but all others are debug?

Comment (ed-sparkes) [reply application/event_processor/changes.py:68]: yeah prob should stay as debug and instead make debug logs on in lower environments
main annoyance was that i couldnt see these logs when i needed to as the log level was set to info

**Review by JackPlowman — APPROVED** [review]
LGTM 👍

**Review by mabe13 — APPROVED** [review]

**Pull request merged**